### PR TITLE
demo/bar-stacked-dataset-update

### PR DESCRIPTION
**Pull request by tordvv**
Updated the dataset of the demo to the latest available dataset from the same source.
Messi and Ronaldo are no longer playing in Champions League. Should we maybe find a better dataset?

### CODE REVIEW

**Review by TorsteinHonsi — CHANGES_REQUESTED** [review]
> Messi and Ronaldo are no longer playing in Champions League. Should we maybe find a better dataset?

Yes, I think that's a good idea

**Comment by tordvv** [comment]
Dataset changed to Formula One constructor standings. I feel this dataset highlights the features of the stacked bar chart well.

File: samples/highcharts/demo/bar-stacked/demo.js [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
         type: 'bar'
     },
     title: {
-        text: 'UEFA CL top scorers by season'
+        text: 'Ferry passengers by vehicle-type 2024'
     },
     xAxis: {
-        categories: ['2020/21', '2019/20', '2018/19', '2017/18', '2016/17']
+        categories: [
+            'Mai', 'April', 'Mars', 'Februar', 'Januar'
```
Comment (TorsteinHonsi) [inline samples/highcharts/demo/bar-stacked/demo.js:11]: Use English month names. 

Also, it looks kind of unintuitive that the months are listed in descending order. Actually I think that the visualization too is better if it starts with January on top. We are used to reading top down.

**Review by TorsteinHonsi — CHANGES_REQUESTED** [review]
Thanks, I like the ferry traffic example. In this dataset, the sum also makes good sense.

File: samples/highcharts/demo/bar-stacked/demo.js (hunk 1 above)
Comment (tordvv) [reply samples/highcharts/demo/bar-stacked/demo.js:11]: Followed the format of the demo. Will update.

**Comment by tordvv** [comment]
Requested changes made

**Review by TorsteinHonsi — APPROVED** [review]

**Pull request merged**